acquisitions/src/main/resources/thunderjet/cross-modules/cross-modules.feature

```diff
@@ -153,6 +153,9 @@ Feature: cross-module integration tests
   Scenario: Pending payment update after encumbrance deletion
     Given call read('features/pending-payment-update-after-encumbrance-deletion.feature')
 
+  Scenario: Remove fund distribution after rollover from open order with re-encumber flag is false
+    Given call read('features/remove_fund_distribution_after_rollover_when_re-encumber_false.feature')
```

Comment:
Feature files are usually using hyphens in files names, and we should keep it consistent.

Reply:
Thank you. Renamed

Reply:
Sorry, I meant you should replace the underscores with hyphens.

Reply:
remove-fund-distribution-after-rollover-when-re-encumber-false

Reply:
Done